tests/unit/test_xasprintf.c

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-FileCopyrightText: 2023, Alejandro Colomar <alx@kernel.org>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+
+#include <stddef.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <stdarg.h>  // Required by <cmocka.h>
+#include <stddef.h>  // Required by <cmocka.h>
+#include <setjmp.h>  // Required by <cmocka.h>
+#include <stdint.h>  // Required by <cmocka.h>
+#include <cmocka.h>
+
+#include "sprintf.h"
+
+
+/**********************
+ * TEST
+ **********************/
+static void
+test_xasprintf(void **state)
```

Comment:
Rename this test case to `test_xasprintf_happy` or `test_xasprintf_ok` or `test_xasprintf_correct`.

Reply:
Ok